### ISSUE TITLE
Add selection info on widget hud

### ISSUE THREAD

**Issue by OrenjiAkira** [uspgamedev/backdoor#452]
Proposal: when you hold the extended_hud state, you can navigate your widgets to display more info about them (and so we can subtiture their textual info with visual info)

**Comment by Kazuo256** [comment]
No, this one refers to the "general" hud that shows up when you hold down `q`. That issue was talking about the widget selection menu used for activating abilities.

Either way, we are rethinking the whole UI, so both might become obsolete.

**Comment by OrenjiAkira** [comment]
A helper for displaying info about the widget would still be useful, tho

**Comment by Kazuo256** [comment]
Yes, but the whole "extended hud state" might not be there anymore, so the specific issue here wouldn't exist anymore. We could always repurpose it though.
